src/styles/utils/variables.scss

```diff
@@ -0,0 +1,12 @@
+$column_grid_count: 4;
+$starscount: 5;
+$width_card: 200px;
+$color__font_secondary: #616070;
```

Comment:
it isn't necessary add `color__` just `$font_secondary`

Reply:
Ready)